target/linux/ath79/dts/ar9344_dlink_dir-825-c1.dts

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar9344_dlink_dir-8x5.dtsi"
+
+/ {
+	model = "D-LINK DIR-825 C1";
+	compatible = "dlink,dir-825-c1", "qca,ar9344";
+
+	aliases {
+		led-boot = &orange_power;
+		led-failsafe = &orange_power;
+		led-running = &blue_power;
+		led-upgrade = &orange_power;
+		serial0 = &uart;
+	};
+
+	leds-1 {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&free_up_usb_led_gpio>;
+
+		blue_power: blue_power {
+			label = "d-link:blue:power";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+
+		blue_usb {
+			label = "d-link:blue:usb";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port1>;
+		};
+
+		/* In /etc/board.d/03_gpio_switches GPIO 20 is turned
+		 * on to cause the WAN LED to blink automatically, so
+		 * the blue WAN LED is not defined here (GPIO 18). */
+
+		blue_wlan2g {
+			label = "d-link:blue:wlan2g";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		blue_wps {
+			label = "d-link:blue:wps";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		orange_power: orange_power {
+			label = "d-link:orange:power";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		orange_wan {
+			label = "d-link:orange:wan";
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds-2 {
+		compatible = "gpio-leds";
+
+		blue_wlan5g {
+			label = "d-link:blue:wlan5g";
+			gpios = <&ath9k 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+	};
+};
+
+&pinmux {
+	free_up_usb_led_gpio: pinmux_free_up_usb_led_gpio {
```

Comment:
use a better name here. look at the naming pattern inside the other DTS files. also the comment can probably be dropped. the directive is self explanatory once correctly named.

Reply:
I changed it to "enable_gpio_11".